tutorial/debezium-vitess-init/local/create_tables_unsharded_customer.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE my_product_seq
+(
+    id      INT,
+    next_id BIGINT,
+    cache   BIGINT,
+    PRIMARY KEY (id)
+) comment 'vitess_sequence';
+
+INSERT INTO my_product_seq(id, next_id, cache)
+VALUES (0, 1000, 100);
+
+CREATE TABLE my_order_seq
+(
+    id      INT,
+    next_id BIGINT,
+    cache   BIGINT,
+    PRIMARY KEY (id)
+) comment 'vitess_sequence';
+
+INSERT INTO my_order_seq(id, next_id, cache)
+VALUES (0, 1000, 100);
+
+CREATE TABLE my_customer_seq
```

Comment:
That's interesting, so no actual sequences in Vitess then?

Reply:
AFAIK, Vitess uses its own implementation of sequence, which is assisted by such sequence tables, to guarantee global uniqueness of incremental keys. Some more documentation can be found here: https://vitess.io/docs/reference/features/vitess-sequences/

Reply:
Ok, gotcha. Thx!